networkx/algorithms/community/quality.py

```diff
@@ -146,7 +148,7 @@ def performance(G, partition):
        Use `partition_quality` instead.
 
     The *performance* of a partition is the ratio of the number of
-    intra-community edges plus inter-community non-edges with the total
+    intra-community edges plus inter-community non-edges divided by the total
```

Comment:
The original wording had "the ratio of" which implies the division of the two quantities. I personally think your wording choice is better, but it should also include removing the "the ratio of" from the previous line.

Reply:
Yes you are right, I fixed it thanks